training/distributed_training/pytorch/model_parallel/mask-rcnn/smp_maskrcnn_tutorial.ipynb

```diff
@@ -0,0 +1,649 @@
+{
```

Comment:
<p>Use <code>hyperparameters</code>  to control what percent of data to train and how many epochs to train. Default to small numbers so that it can be tested </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1965/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>